types/twine-sugarcube/test/ui.ts

```diff
@@ -0,0 +1,93 @@
+//#region  Dialog
+
+const dialogOptions: TwineSugarCube.DialogOptions = {
+	top: 123,
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/twine-sugarcube/test/ui.ts;
  Line: 4;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.